pymodbus/client/base.py

```diff
@@ -80,6 +80,7 @@ def __init__(
                 parity=kwargs.get("parity", None),
                 stopbits=kwargs.get("stopbits", None),
                 handle_local_echo=kwargs.get("handle_local_echo", False),
+                rs485_settings=kwargs.get("rs485_settings", None),
```

Comment:
Parameter doc missing, please see serial.py line 35++

Since this is a serial parameter only, it should be moved to serial.py

right now you are adding rs485_settings to all types.

In general it should be a line in comm_params.

Reply:
done